infrastructure/server-setup-checklist.md

```diff
@@ -0,0 +1,18 @@
+# Server setup checklist
```

Comment:
I created this list to help with deployments as there starts to be quite a bit that needs checking after first deployment